[Durable Objects] add an example GraphQL query (#9097)
cloudflare/cloudflare-docs: content/workers/learning/using-durable-objects.md

### GraphQL Analytics

Durable Object metrics are powered by GraphQL, like other Workers metrics. Learn more about querying Workers data sets in this [tutorial](/analytics/graphql-api/tutorials/querying-workers-metrics/). The data sets that include Durable Object metrics include `durableObjectsInvocationsAdaptiveGroups`, `durableObjectsPeriodicGroups`, `durableObjectsStorageGroups`, and `durableObjectsSubrequestsAdaptiveGroups`. You can [use GraphQL introspection to get information on the fields exposed by each](/analytics/graphql-api/getting-started/explore-graphql-schema).

> # An example of GraphQL query for Durable Objects

Reviewer: ```suggestion
### Example of GraphQL query for Durable Objects
```

Durable Object metrics are powered by GraphQL, like other Workers metrics. Learn more about querying Workers data sets in this [tutorial](/analytics/graphql-api/tutorials/querying-workers-metrics/). The data sets that include Durable Object metrics include `durableObjectsInvocationsAdaptiveGroups`, `durableObjectsPeriodicGroups`, `durableObjectsStorageGroups`, and `durableObjectsSubrequestsAdaptiveGroups`. You can [use GraphQL introspection to get information on the fields exposed by each](/analytics/graphql-api/getting-started/explore-graphql-schema).

> 

Reviewer: ```suggestion
# An example of GraphQL query for Durable Objects

Refer to the [Querying Workers Metrics with GraphQL](/analytics/graphql-api/tutorials/querying-workers-metrics/) tutorial for authentication

```

```

> # An example GraphQL query for Durable Objects

Reviewer: ```suggestion
# An example of GraphQL query for Durable Objects
```

> # An example of GraphQL query for Durable Objects

Reviewer: ```suggestion
```

> # Refer to the tutorial linked above for authentication

Reviewer: ```suggestion
Refer to the [Querying Workers Metrics with GraphQL](/analytics/graphql-api/tutorials/querying-workers-metrics/) tutorial for authentication
```

> Refer to the [Querying Workers Metrics with GraphQL](/analytics/graphql-api/tutorials/querying-workers-metrics/) tutorial for authentication

Reviewer: ```suggestion

```

> {

Reviewer: ```suggestion
```
{
```
